tests/python/unittest/test_meta_schedule_relay_tir_compute.py

```diff
@@ -147,8 +147,17 @@ def schedule_fn(task, sch):
         return False
 
     with TempOpAttr("nn.conv2d", "FTVMStrategy", _tmp_strategy):
-        database = apply_fixed_schedules(relay_mod, target, params, schedule_fn)
-        with ApplyHistoryBest(database):
+        database = apply_fixed_schedules(
+            relay_mod,
+            target,
+            params,
+            schedule_fn,
+            te_filter_func="meta_schedule.DefaultTaskFilterAllowExtern",
```

Comment:
@csullivan there are a couple of options in my mind for us to unbreak MS's current workflow with customer models:
- use `te_filter_func` when we want to specifically allow some `te.extern` operation;
- use a different `te.extern_prim_func` to direct the default task filter to accept them by default
- use annotations to instruct the default task filter instead 
let me know what you think!

Reply:
@junrushao1994 thanks for the discussion! I think the approach you have taken with a separate filter function is a good solution for now. 
From the integration side I think it could indeed be desirable to separate out the functionality into a new TE Op (your second option I believe) that can be included by the default filter func so that it is transparent to end user. But for now I think it's perfectly fine to approach it when the need arises.  Thanks!